acceptance/apps_test.go

```diff
@@ -227,4 +228,60 @@ var _ = Describe("Apps", func() {
 			}, "1m").Should(MatchRegexp(`Status .*\|.* 1\/1`))
 		})
 	})
+
+	Describe("logs", func() {
+		var (
+			appName, route string
+			logLength      int
+		)
+
+		BeforeEach(func() {
+			appName = newAppName()
+			out := makeApp(appName, 1, true)
+			routeRegexp := regexp.MustCompile(`https:\/\/.*omg.howdoi.website`)
+			route = string(routeRegexp.Find([]byte(out)))
+
+			out, err := Epinio("app logs "+appName, "")
```

Comment:
From here on the code feels again to be something more for `It` instead of `BeforeEach` ?!

Reply:
In this case I think it's fine. This code simply calculates the number of lines already printed in order to be used later in the test. It is not testing anything in the BeforeEach block (except for unexpected error handling).